docs/src/main/sphinx/connector/object-storage-file-formats.rst

```diff
@@ -0,0 +1,94 @@
+===========================
+Object storage file formats
+===========================
+
+Object storage connectors support one or more file formats specified by the
+underlying data source.
+
+In the case of serializable formats, only specific
+`SerDes <https://www.wikipedia.org/wiki/SerDes>`_ are allowed:
+
+* RCText - RCFile ``ColumnarSerDe``
```

Comment:
These might be wrong now with the refactor .. but I think we should merge like this and then improve more after